public/docs/ts/_cache/tutorial/toh-pt6.jade

```diff
@@ -115,15 +126,24 @@ block dont-be-distracted-by-backend-subst
 
 :marked
   We returned a !{_Promise} resolved with mock heroes.
-  It may have seemed like overkill at the time, but we were anticipating the
+  It may have seemed like an overkill at the time, but we were anticipating the
```

Comment:
Interesting. It seems that [I wasn't the only one surprised by this](http://english.stackexchange.com/questions/100630/seems-like-an-overkill-vs-seems-like-overkill) ... but I understand now.

Fixed.